tools/generate_linux_syscalls.zig

```diff
@@ -233,10 +259,7 @@ pub fn main() !void {
     // header file for their syscall numbers. Arch-specific headers are used to
     // define pre-proc. vars that add additional (usually obsolete) syscalls.
     //
-    // TODO:
-    // - It would be better to use libclang/translate-c directly to extract the definitions.
-    // - The `-dD` option only does minimal pre-processing and doesn't resolve addition,
-    //   so arch specific syscalls are dealt with manually.
+    // TODO: It would be better to use libclang/translate-c directly to extract the definitions.
```

Comment:
wishful thinking. just delete the todo comment. file an issue if you want. or don't